src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayRent = 40;
+  const discountFor7Days = 50;
+  const discountFor3Days = 20;
+  const minDaysFor7DayDiscount = 7;
+  const minDaysFor3DayDiscount = 3;
```

Comment:
Constants should be in UPPER_CASE, and you can move them outside the function
<img width="356" alt="image" src="https://github.com/user-attachments/assets/a6a60af0-f4ab-46a3-ab9c-0c9113be33bc">